<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>THandEyeCali2D</class>
 <widget class="QWidget" name="THandEyeCali2D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>THandEyeCali2D</string>
  </property>
  <widget class="QLabel" name="label_IR_R">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>480</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_gain">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>曝光时间(us)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_show_TCP">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>show TCP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_exposure_time">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>30000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_IR_L">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>480</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>增益</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_calculate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>510</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_capture">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>510</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>拍照提取角点</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_get_IR">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>510</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>仅拍照</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
